scripts/pylintrc_default

```diff
@@ -200,7 +200,7 @@ no-space-check =
 # Maximum number of lines in a module
 # DEFAULT:  max-module-lines=1000
 # RATIONALE:  API-mapping
-max-module-lines=1593
+max-module-lines=1200
```

Comment:
Doesn't the absurdity of tweaking such arbitrary numbers ever make you question their value?